modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/samples/wsse/policy/basic/SignEncryptGCMTestCase.java

```diff
@@ -45,11 +45,12 @@
  * @author alessio.soldano@jboss.com
  * @since 27-Feb-2012
  */
-@RunWith(Arquillian.class)
+@ExtendWith(ArquillianExtension.class)
+//https://issues.jboss.org/browse/JBEAP-5200
+@DisabledIfSystemProperties({@DisabledIfSystemProperty(named = "java.vendor", matches = "IBM Corporation"),
```

Comment:
indeed, we are running JDK11+, will remove this line.

Reply:
I think we'd better keep this. As we don't run IBM JDK 11 , this can probably gives use some hints that this isn't used to work with IBM JDK8, probably this still doesn't work with IBM JDK11 either.